metadata_writer_sqlite.h

```diff
@@ -65,6 +65,7 @@
                             "NetworkProvider INT," \
                             "NetworkAddressFamily INTEGER NOT NULL," \
                             "NetworkAddress TEXT NOT NULL," \
+                            "Type INTEGER NOT NULL," \
```

Comment:
We don't dump Type in case of SQL format. We dump this column only when JSON output format is selected. The reason why I added it is just simpler impementation, because we can handle everything in SQL statement. No need to check table name and convert it to type. So "Type" is only in JSON, not in SQL output format.